ql/src/java/org/apache/hadoop/hive/ql/util/TimeUtil.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hive.ql.util;
+
+/**
+ * Makes it possible to mock test your code.
+ */
+public class TimeUtil {
```

Comment:
Why is this class required? For mocking? Can't we use the existing util class like ```Time.monotonicNowNanos()```? rather than introducing our own

Reply:
Thx for the tip. I didn't know that we have this. As I see it is in apache-common and it is already a dependency of Hive.